### DISCUSSION TITLE
!PB/KC Chat commands for colossal wyrm

### DISCUSSION THREAD

**Discussion by Bass-Music** [runelite/runelite#18545] [Ideas]
Can we add chat commands for colossal wyrm (basic/advanced) and helpful shortcut abbrv's? 

!Pb colossal wyrm (basic)
!Pb colossal wyrm (advanced)

Thank you!

**Comment by Felanbird** [comment]
These already exist but since they use Jagex's official names by default, they are `!pb colossal wyrm agility course (basic)` and advanced

**Reply by Bass-Music** [reply]
Thanks for the quick reply. Perhaps some abbreviations to match Shayzien would be helpful -

wyrmbac, wyrmaac, wyrm basic/advanced, colossal wyrm basic/advanced, wyrmbasic/advanced, col wyrm basic/advanced, colwyrm basic advanced -> colossal wyrm agility course (basic/advanced)


shayb, sbac, shayzienbasic, shayzien basic -> Shayzien Basic Agility Course

**Comment by Nightfirecat** [comment]
Added in 8da309b890f249a5a74d83cb96ee7582ca16a141
